app/Jobs/ProcessSubmission.php

```diff
@@ -204,7 +207,7 @@ private function doSubmit($filename, $projectid, $buildid = null, $expected_md5
         }
 
         // Parse the XML file
-        $handler = ctest_parse($filehandle, $projectid, $expected_md5, $buildid);
+        $handler = ctest_parse($filehandle, $filename, $projectid, $expected_md5, $buildid);
```

Comment:
It looks like this `$filename` argument isn't used anymore

Reply:
We use it to build an error message if a type isn't found (https://github.com/Kitware/CDash/pull/2437/files#diff-098ba7a804e6a9e90757b2ccdc214de5260fc409564e3778ce806d1a13e01e54R83).  

Should we just drop that part or use something like: https://stackoverflow.com/questions/5144583/getting-filename-or-deleting-file-using-file-handle/7690726#7690726 to ask for the `$filehandle` name?